about.html

```diff
@@ -0,0 +1,125 @@
+        </p>
+      </div>
+      <div class="contact">
+        <p>
+          Please contact us per Email for any further questions about our First Interplanetary Arrival Conference 2022!
+        </p>
+        <p class="email">fiacs2022@globalsecurity.org</p>
+      </div>
+    </header>
+    <section class="about-logo">
+      <div>
+        <h2 class="the-logo">First Interplanetary Arrival Conference Logo</h2>
+      </div>
+      <hr class="logo-hr" />
+      <div class="text">
+        The logo of the First Interplanetary Arrival Conference was imposed by us by the Legal Ambassador of Atlkaya-Mirbukia Tuloska Galaxy Ring.
+      </div>
+      <div class="logo">
+        <img src="./images/FIAC-logo-2X.png" alt="FIAC logo" />
+      </div>
+    </section>
+    <hr class="between-sections" />
+    <section class="about-planning">
+      <div>
+        <h2 class="possible">How this event was made possible?</h2>
+      </div>
+      <hr class="planning-hr" />
+      <div class="text">
+        Take a look at the planning that went into the making of the one in a
+        lifetime event. First contact with extraterrestrial beings is no easy
+        task to plan!
+      </div>
+      <div class="videos-wrapper">
+        <div class="video v1">
+          <h2>First Contact</h2>
+          <p>When SETI Contacted NASA and Everything Began</p>
+        </div>
+        <div class="video v2">
+          <h2>UNSC Communication</h2>
+          <p>United Nations Security Council Planning with Intergallactic Beings</p>
+        </div>
+      </div>
+    </section>
+    <section class="organizers">
+      <h2>Organizers</h2>
+      <hr class="organizers-hr" />
+      <div class="organizers-wrapper">
+        <img
+          src="./images/sponsor-logos/china_national_space_administration_logo.png"
+          alt=""
+        />
+        <img src="./images/sponsor-logos/esa-logo.png" alt="european space agency" />
+        <img src="./images/sponsor-logos/nasa-logo.png" alt="nasa" />
+        <img src="./images/sponsor-logos/seti-institute-logo.png" alt="seti" />
+        <img src="./images/sponsor-logos/spacex-logo.png" alt="spacex" />
+        <img
+          src="./images/sponsor-logos/united_states_space_force_logo.png"
+          alt="us air force"
+        />
+        <img src="./images/sponsor-logos/unsc-logo.png" alt="united nations security council" />
+      </div>
+    </section>
+    <footer>
+      <div class="logo">
+        <img class="logo-img" src="./images/FIAC-logo-2X.png" alt="FIAC logo" />
+      </div>
+      <div class="copy">
+        <p>
+          2022 First Interplanetary Arrival Conference. Some Rights Reserved.
+        </p>
+      </div>
+    </footer>
```

Comment:
- [ ] [OPTIONAL] Hi, Well done on the design of your website. I'd like to point out that the desktop version of the about page should have a dark background according to the Figma design, I can see on yours you have a White background